resources/views/welcome.blade.php

```diff
@@ -1,27 +1,29 @@
 <!DOCTYPE html>
 <html>
 
+<head>
+    <title>Welcome to Lucy!</title>
+    <link rel="stylesheet" href="/css/app.css">
+</head>
+<body class = "welcome-body">
 
-<body style="background-color:lightsteelblue">
-<center>
-    <h1 style = "padding-top: 30px">
-        <p style="font-size:2vw">Welcome to</p>
-        <p style="font-size:3vw">Project Lucy</p>
-    </h1>
-    <p>An Augmentative and Alternative Communication Open Source Solution</p>
+    <div class = "welcome-title">
+        <p class = "welcome-header">Welcome to</p>
+        <p class = "welcome-header">Project Lucy</p>
+        <p>An Augmentative and Alternative Communication Open Source Solution</p>
+    </div>
     <br>
     <br>
-    <h1>
-        <button style="width: 200px; height: 80px; background-color:lightgoldenrodyellow;">
-        <font size="+1"><a href="http://localhost:8080/login">Log in</a></font>
-        </button>
-        <button style="width: 200px; height: 80px; background-color:lightgoldenrodyellow;">
-        <font size="+1"><a href="http://localhost:8080/create-account">Create Account</a></font>
-        </button>
-        <button style="width: 200px; height: 80px; background-color:lightgoldenrodyellow;">
-        <font size="+1"><a href="http://localhost:8080">Guest</a></font>
-        </button>
-    </h1>
-    
-</center>
+    <div>
+        <a href="http://localhost:8080/login">
+            <button class = "welcome-button">Log in</button>
+        </a>
+        <a href="http://localhost:8080/create-account">
+            <button class = "welcome-button">Create Account</button>
+        </a>
+        <a href="http://localhost:8080">
+            <button class = "welcome-button">Guest Login</button>
```

Comment:
totally a nitpick but I don't think we need to have ` Login` here, it can still just be `Guest` since they won't have to log in at all

Reply:
okay I'll change it